charts/aws-ebs-csi-driver/templates/NOTES.txt

```diff
@@ -1,3 +1,43 @@
 To verify that aws-ebs-csi-driver has started, run:
 
     kubectl get pod -n {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "aws-ebs-csi-driver.name" . }},app.kubernetes.io/instance={{ .Release.Name }}"
+
+
+WARNING: The following values have been deprecated in favor of moving them into the controller or node groups.  They will be removed in a subsequent release.
```

Comment:
this is great.

Reply:
Thanks, I though it was a decent way to handle it